### PR TITLE
fix: [FFM-6735]: Fixing issue with feature metrics search and remove feature flag

### PR DESCRIPTION
**Pull request by chrisblakely01**
### Summary

Fixes an issue with metric search and removes feature flag

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Sonar: `retrigger sonar`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Feature Name Check: `trigger featurenamecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress Rest: `retrigger cypress-rest`
- Cypress CD: `retrigger cypress-cd`
- Cypress Pipeline: `retrigger cypress-pipeline`
- Cypress CV: `retrigger cypress-cv`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

**Comment by chrisblakely01** [comment]
retrigger titlecheck

**Comment by chrisblakely01** [comment]
retrigger coverage

**Comment by chrisblakely01** [comment]
retrigger coverage

File: src/modules/75-cf/pages/feature-flags/components/FeatureFlagsListing.tsx [hunk 1]
```diff
@@ -50,6 +49,7 @@ const FeatureFlagsListing: React.FC<FeatureFlagsListingProps> = ({
   features,
   toggleFeatureFlag,
   featureMetrics,
```
Comment (knagurski) [inline src/modules/75-cf/pages/feature-flags/components/FeatureFlagsListing.tsx:51]: If you default this to an empty array, you can simplify the code below and know it'll be an array

File: src/modules/75-cf/pages/feature-flags/FeatureFlagsPage.tsx [hunk 2]
```diff
@@ -85,38 +83,12 @@ const FeatureFlagsPage: React.FC = () => {
...
-            [flagFilter.queryProps?.key]: flagFilter.queryProps?.value
-          }
-        })
-
-        setFeatureMetrics(metrics as FeatureMetric[])
-      }
+  const { data: featureMetrics, loading: featureMetricsLoading } = useGetFeatureMetrics({
+    queryParams: {
+      ...queryParams,
+      environmentIdentifier: environmentIdentifier
```
Comment (knagurski) [inline src/modules/75-cf/pages/feature-flags/FeatureFlagsPage.tsx:89]: Deffo not a blocker, but you can simplify
```suggestion
      environmentIdentifier
```

**Review by crystal-fong — DISMISSED** [review]
tested this in the UI, seems good 👍

**Review by knagurski — APPROVED** [review]

**Comment by knagurski** [comment]
Actually, before you merge, can we delete the flag itself? `FFM_6610_ENABLE_METRICS_ENDPOINT`

**Review by knagurski — APPROVED** [review]
🔥

**Comment by chrisblakely01** [comment]
retrigger cypress-cd

**Review by crystal-fong — APPROVED** [review]

**Pull request merged**